### PR TITLE
add a new blog

### PR DESCRIPTION
**Pull request by husiyu**

### CODE REVIEW

File: geographic-data-distribution-traffic-and-latency-halved.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+<center>图 3 主数据中心 & 从数据中心部署</center>
+
+### Follower Read Improvement
+
+在 TiDB 里面，当我们需要从西安这边读取数据的时候，一个典型的流程如下：
+
+1.  西安的 TiDB 向北京的 PD 发起获取 TSO 请求，得到一个 start_ts（事务开始阶段的 ID）。（1 RTT）
+
+2.  西安的 TiDB 为涉及到的每个 region 向北京的 TiKV leader 节点发起多个（并行）读请求（如图 4）。（1 RTT）
```
Comment (hicqu) [inline geographic-data-distribution-traffic-and-latency-halved.md]: s/region/Region

File: geographic-data-distribution-traffic-and-latency-halved.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+*   [https://github.com/tikv/raft-rs/pull/249/](https://github.com/tikv/raft-rs/pull/249/)
+
+*   [https://github.com/etcd-io/etcd/issues/11357](https://github.com/etcd-io/etcd/issues/11357)
+
+## 总结
+
+除了我们在 Hackathon 做的两个优化，跨数据中心的场景有更多需要解决的问题和可以优化的点，我们的优化也远非最终实现，一些不难想到的优化还有：
+
+1.  Follower Read Improvement 能将一个非交互式的读事务从 2RTT 降到 1RTT，但对于交互式的读事务，由于事先不知道涉及到事务的 Region，无法预读整个读请求中所有 region `read_index`，因此只有第一次读请求和 `get_tso` 可以并行，将 n+1 RTT 优化到了 n RTT（n 为交互式事务中读语句的数量），而如果我们能将 ts 和 committed index 的对应关系找到，并且定期维护每个 region 的 safe ts（小于该 ts 的事务一定已经 committed or aborted），那么我们就可以将交互式读事务的延迟也降低到 1RTT。
```
Comment (hicqu) [inline geographic-data-distribution-traffic-and-latency-halved.md]: s/region/Region

**Comment by husiyu** [comment]
@YiniXu9506 PTAL

**Comment by YiniXu9506** [comment]
LGTM

**Pull request merged**